ci-operator/step-registry/baremetal/lab/agent/install/network/static/day2/baremetal-lab-agent-install-network-static-day2-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+workflow:
+  as: baremetal-lab-agent-install-network-static-day2
+  steps:
+    pre:
+      - chain: baremetal-lab-upi-conf
```

Comment:
if I understand correct, we shouldn't use `- chain: baremetal-lab-upi-conf` , rather `- chain: baremetal-lab-agent-conf`.
in your case, the additional node is a normal arm node, rather than an agent type arm64 node.

Reply:
as we discuessed privately, @zniu1011 share me agent ISO image needs to contains all nodes info and is One-time configuration in the installtion, we can't create ISO image for day2 operators.  day2 node can't boot up with agent way.